pages/common/mu.md

```diff
@@ -0,0 +1,30 @@
+# mu
+
+> Index and search emails from a local Maildir.
+> More information: <https://www.djcbsoftware.nl/code/mu/>.
```

Comment:
Although the given link is from official page, but the page only contain a link of a cheat sheet rather than detailed manual. I'd prefer to use the man page from https://manned.org/mu.
```suggestion
> More information: <https://manned.org/mu>.
```

Reply:
You're right, apart from fairly hidden details about `mu init` which are not in the man page, the manual there is only about mu4e.

Reply:
Actually, this man page was last updated in 2013, so before the 0.9.6 version introducing `mu init` and some other changes. If it's fine to link to any online man page, https://man.cx/mu(1) seems to have been updated this month. I should have checked that before committing.